packages/mdc-menu/index.js

```diff
@@ -120,7 +120,7 @@ class MDCMenu extends MDCComponent {
   }
 
   /**
-   * @param {!AnchorMargin} margin
+   * @param {!MenuPosition} margin
```

Comment:
I suspect this is going to need updating... and will this even pass closure compiler tests right now given that menu-surface is converted but this isn't?

Reply:
No idea, but we removed the Closure Compiler test from the `feat/typescript` branch.

Reply:
I had forgotten there are no longer closure compiler tests on this branch, so this is fine. I was mainly concerned about CI being able to pass.